docs/community/contributing.md

```diff
@@ -85,20 +84,13 @@ or Eventing and chat with other project developers. See also the [Knative Slack
 - [Stack Overflow questions](https://stackoverflow.com/questions/tagged/*knative*):
 Knative tagged questions and curated answers.
 
-## Community Meetup
+### Community Meetups
 
-This virtual event is designed for end users, a space for our community to meet,
-get to know each other, and learn about uses and applications of Knative.
+New to Knative or built something cool?  Share your impressions by
+presenting at the Knative Community Meetup!
```

Comment:
do we have a link on how to do volunteer to present? (coincidentally was looking for this info earlier and couldn't find it)

Reply:
found it: https://t.co/p7qiU4Dob9

Reply:
Done